table/sst_file_reader.cc

```diff
@@ -113,7 +113,7 @@ std::unique_ptr<Iterator> SstFileReader::NewTableIterator() {
     // InternalIterator.
     return nullptr;
   }
-  return std::make_unique<TableIterator>(internal_iter);
+  return std::make_unique<TableIterator>(internal_iter, &rep_->options);
```

Comment:
@ajkr @pdillinger Can you comment on this change in the internals for RocksDB please?